### PR TITLE
Adding community and newsletter to LocalFirstWeb.dev (LFW.dev)

### PR DESCRIPTION
**Pull request by yfeleke**
Discord community with button on top level page: https://discord.gg/AUjyVxUbCH 
Added a community section under Join In.
Also https://lfw.dev is now redirecting to localfirstweb

### CODE REVIEW

File: _data/content.json [hunk 1]
```diff
@@ -169,6 +169,12 @@
             "author": "@gaforres",
             "url": "https://twitter.com/gaforres",
             "icon": "https://pbs.twimg.com/profile_images/1350947174645956609/Ebe_Qd46_200x200.jpg"
+          },
+          {
+            "title": "James Pearce",
+            "author": "@jamespearce",
```
Comment (yfeleke) [inline _data/content.json:175]: I got you @jamesgpearce  :)

**Review by jamesgpearce — CHANGES_REQUESTED** [review]
Cool but this section was more about examples of local-first apps you can use. Maybe we should have a section call 'Discuss' or something?

**Comment by yfeleke** [comment]
Will update and resend

**Comment by jamesgpearce** [comment]
Are you ok resolving the conflicts? We've had a few additional entries come in overnight.

**Comment by yfeleke** [comment]
> Are you ok resolving the conflicts? We've had a few additional entries come in overnight.

Happy to, I'll rebase

**Pull request closed without merging**